### ISSUE TITLE
Customizable save path?

### ISSUE THREAD

**Issue by Stef-00012** [gbl/GrabcraftLitematic#7]
is the save path of the new .litematic file customizable?

**Comment by gbl** [comment]
No, it always saves to the "schematics" folder, which is where Litematica reads its schematics from.
